pages/review.vue

```diff
@@ -154,16 +151,18 @@ export default {
     sendMail() {
       const data = {
         deadline: this.formattedDate,
-        tonesCount: this.tonesCount,
-        soundsCount: this.soundsCount,
         user: this.meta.user,
         title: this.meta.title,
         format: this.meta.format,
-        speaker: this.meta.speaker
+        speaker: this.speaker.name,
+        comment: this.comment
       }
 
-      postEmailApi(data)
-      postOrderEmailApi(data)
+      fetch('/api/orders', {
```

Comment:
We still have the data in the cookie, right? We need the same device+browser+cache when we click on the link?

Reply:
Not sure what you mean?

Reply:
I thought about clicking on a link when you open an email: You need to open the same browser, you cannot click on the link e.g. on your mobile.